libs/ui/ui-shared/src/lib/components/Table/Table.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { UserType, OrderType, ProductType } from '../../../types';
+import Button from '../Button/Button';
+import styles from './Table.module.scss';
+import { MdEdit, MdDelete } from 'react-icons/md';
+
+export type TableProps<T> = {
+  admin: boolean;
+  header: string[];
+  content: T[];
+};
+
+export const Table = <T extends UserType | OrderType | ProductType>({
```

Comment:
Your table should not depends on your domain types, check https://ant.design/components/table and use it